src/roles/common/templates/group_vars.all.j2

```diff
@@ -49,6 +49,19 @@ webfilter_cpu_cores: {{ common.webfilter_cpu_cores | default(2) }}
 vcenter_start_vsd_with_interfaces_disabled: {{ common.vcenter_start_vsd_with_interfaces_disabled | default(false) }}
 vsd_inplace_upgrade: {{ inplace | default(false) }}
 
+
+{% if encrypted.nuh_notification_app_1_username is defined and encrypted.nuh_notification_app_1_password is defined %}
+notification_app1:
+  username: {{ encrypted.nuh_notification_app_1_username }}
+  password: {{ encrypted.nuh_notification_app_1_password }}
+{% endif %}
+
+{% if encrypted.nuh_notification_app_2_username is defined and encrypted.nuh_notification_app_2_password is defined %}
+notification_app2:
+  username: {{ encrypted.nuh_notification_app_2_username }}
```

Comment:
You will need the filter "| indent(8, False)" like other encrypted fields.

Reply:
done